src/app/pages/ArticlePage/index.jsx

```diff
@@ -59,11 +61,7 @@ const StyledMain = styled.main`
   flex-grow: 1;
 `;
 
-const MostReadSection = styled.section.attrs(() => ({
-  role: 'region',
-  'aria-labelledby': 'Most-Read',
-  'data-e2e': 'most-read',
-}))`
+const ArticlePageMostReadSection = styled(MostReadSection)`
```

Comment:
Is there any way we can avoid having this and `FrontPageMostReadSection`? Perhaps I'm not understanding it correctly but I'm wondering if this will apply for all future page types we make where we'll need a specific section made just to add Most Read to it.

Reply:
A11y requirements specify that most-read should be in its own section. The styling for the most read section in the `ArticlePage` is different from that on the `FrontPage` even though the attributes applied to these sections are the same.